### ISSUE TITLE
[White Label - Card Wizard]: Narrator is not announcing the label information for Name button.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5720]
### What is the PWD impact?

Screen reader users are unable to understand the purpose of the Name button, preventing them from interacting independently with the UI and potentially blocking task completion.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Pre-Requisite:
Turn on Narrator

Repro Steps
1. Open URL: chro and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type Card Wizard word in type your message text box.
4. Response will appear.
5. Navigate to the Name control.
6. Observe an issue that Narrator announcing incorrect role as toggle button for Name button.

### What do you expect?

The element should expose a programmatically determinable accessible name  so that Narrator clearly announces the label associated with the Name button.
Observation: When focus lands on Name control narrator should announce please provide the following information Name control.

### What actually happened?

Narrator is not announcing the label information for Name button.
Observation: when focus lands on Name button it is announcing as Name toggle button on.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/22f70c90-0aaa-4391-a22f-f9ae528ac926

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

### Adaptive Card JSON

```json

```
